roles/openshift_gcp/templates/yum_repo.j2

```diff
@@ -16,5 +16,9 @@ sslclientkey={{ "/var/lib/yum/custom_secret_" + (loop.index-1)|string + "_key" i
 {% if key not in ['id', 'name', 'baseurl', 'enabled', 'gpgcheck', 'sslclientkey', 'sslclientcert'] and value is defined %}
 {{ key }}={{ value }}
 {% endif %}
+{% if 'user_pass_auth' in repo %}
+username_file=/usr/share/ansible/openshift-ansible/inventory/dynamic/injected/USERNAME
```

Comment:
filenames are like we agreed: `repository-credentials-{username,password}`, just their content is `USERNAME` and `PASSWORD` for now

Reply:
ah - thanks.

Reply:
changed pushed.